examples/layers/services/isolate.dart

```diff
@@ -177,7 +177,8 @@ class CalculationManager {
   //
   // Static and global variables are initialized anew in the spawned isolate,
   // in a separate memory space.
-  static void _calculate(CalculationMessage message) {
+  static void _calculate(Object isolateData) {
```

Comment:
This change seems very unfortunate.

Reply:
Yeah, I hate these too, but that's the only contract that's provided by the call site.  A long time ago I wrote a proposal to allow the equivalent of `covariant` on callbacks like this to allow this.  We could try to resurrect that at some point.

Reply:
Seems like Isolate.spawn should be generic too.

Reply:
@lrhn Are there plans to make Isolate.spawn generic in the SDK?

Reply:
Yes. It is supposedly a non-breaking change that should have been handled with the other strong-mode migrations.
I just haven't gotten around to it yet.

Reply:
@lrhn has a CL out making Isolate.spawn generic in our core libraries.  I'm not sure how this percolates into your SDK.  The relevant file is `flutter/bin/cache/pkg/sky_engine/lib/isolate/isolate.dart` .  How does this get updated?  Do I need to submit a separate pull request somewhere?

@Hixie I'm fine making this one a //ignore if you prefer to paper over the short term period until your copy of isolate.dart gets updated.  Or I can leave it as is and file a bug to revert?